utils/path_manager.py

```diff
@@ -0,0 +1,45 @@
+import os
```

Comment:
rename `path_manager.py` to `paths.py`, and create a `Paths` class, with static methods:

- `.cache`
- `.model_repository`
- `.dataset`
- `.inference`
- `.append_hf_training_file(job_path)`
- `.append_hf_model_folder(job_path)`
- `.append_hf_training_args_folder(model_path)`